app/models/game.rb

```diff
@@ -26,19 +26,26 @@ def self.search(query)
     4 => "extreme"
   }
 
-  validates :name, presence: true, uniqueness: true
+  validates :name, presence: true,
+    uniqueness: true
   validates :user, presence: true
-  validates :description, presence: true
-  validates :min_players, presence: true, numericality: {
+  validates :description, presence: true,
+    length: { minimum: 250 }
+  validates :min_players, presence: true,
+    numericality: {
     less_than_or_equal_to: :max_players,
     message: "must be less than or equal to max players!",
     greater_than_or_equal_to: 1, message: "must be at least 1",
     only_integer: true
   }
-  validates :max_players, presence: true, numericality: {
+  validates :max_players, presence: true,
+    numericality: {
+    greater_than_or_equal_to: 1, only_integer: true
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.